### DISCUSSION TITLE
Pointing telescope: find Alt-Az from RA/Dec, site and time

### DISCUSSION THREAD

**Discussion by fdesvallees** [skyfielders/python-skyfield#782] [Q&A]
Hello: 

This must be very easy, but I keep getting errors like "AttributeError: 'ICRF' object has no attribute 'at'.
Can you please help?

Thank you
François

**Comment by brandon-rhodes** [comment]
We usually need to see some sample code to identify what has gone wrong. Can you share the full traceback? Can you share a short script that re-creates the situation you are seeing, that we can run on our own computers to see the same error and investigate solutions to it? Thanks!

**Comment by fdesvallees** [comment]
Here it is again (slightly modifiled).  This is my current understanding of how it should work, but it tells me:

AttributeError: 'ICRF' object has no attribute '_observe_from_bcrs'

Thank you!


```
from skyfield import api

ts = api.load.timescale()

planets = api.load('de421.bsp')
site = planets['earth'] + api.wgs84.latlon(45, 5)

ra = 16.56
dec = -26

t = ts.now() 

star = api.position_of_radec(ra,dec)
astrometric = site.at(t).observe(star)
apparent = astrometric.apparent()
altaz = apparent.altaz()
print (altaz)

```

**Comment by fdesvallees** [comment]
OK, I fixed it. I was missing a Star() constructor. I found it in another example. Thanks anyway Brandon!

François


```
from skyfield import api 

ts = api.load.timescale()

planets = api.load('de421.bsp')
site = planets['earth'] + api.wgs84.latlon(45, 5)

ra = 16.56
dec = -26

ts = api.load.timescale()
t = ts.now() 

star = api.Star(ra_hours=16.56,dec_degrees=-26)
astrometric = site.at(t).observe(star)
apparent = astrometric.apparent()
altaz = apparent.altaz()
print (altaz)
```

**Reply by brandon-rhodes** [reply]
I'm glad you found a fix and that your script is working!
